README.md

```diff
@@ -1,38 +1,82 @@
-# Newdle
+## WARNING: newdle is not yet production ready
+### Currently missing
+ * [ ] E-mail sending;
+ * [ ] Integration with calendaring systems;
+ * [ ] Integration with Indico;
 
-This is something new and cool! 🎉
+If you would like to help with any of that, please contact us through the relevant GitHub issue.
 
-## Database
+# Newdle [![Travis Build Status](https://travis-ci.com/indico/newdle.svg?branch=master)](https://travis-ci.org/indico/newdle) [![License](https://img.shields.io/github/license/indico/newdle.svg)](https://github.com/indico/newdle/blob/master/LICENSE) [![Made at CERN!](https://img.shields.io/badge/CERN-Open%20Source-%232980b9.svg)](https://home.cern)
+
+<p align="center">
+  <img src="./newdle/client/src/images/logo_color.svg" width="300"><br>
```

Comment:
relative paths here seem very risky

Reply:
Hm, I could either move it to the root path of the repo or to the `assets` repo and reference it by here via URL.